docs-src/go/how-to-list-workflow-executions-using-the-client-in-go.md

```diff
@@ -9,4 +9,31 @@ tags:
   - client
 ---
 
-Use [`Client.ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow).
+The `ListWorkflow()` function retrieves [Workflow Executions](/concepts/what-is-a-workflow-execution) that match the properties specified in a [List Filter](/concepts/what-is-a-list-filter).
+These properties are defined by [Search Attributes](/concepts/what-is-a-search-attribute), which are used to retrieve Workflow Executions from the [Visibility store](/concepts/what-is-visibility).
```

Comment:
Should `Visibility store` link to `/cluster-deployment-guide#visibility-store`?

Reply:
I think the specificity is a little bit off...

WFEs are not stored in the visibility store - they are stored in the normal persistence. Search attribute data is stored in the visibility store (what search attributes are part of the system basically)

WFEs are stored in normal persistence.
Search Attribute source of truth data is in the Vis store.
The ListWorkflow function is used to retrieve a list of WFs
The List Filter provided to the ListWorkflow call filters and sorts the list.
What is returned is WFE meta data - 
You can use that to get the full details and history of and individual WFE.